translations/en/module_en_lang.php

```diff
@@ -52,7 +52,7 @@
     'wd_config_message' => 'Your message',
     'wd_config_payment_action' => 'Payment Action',
     'wd_config_payment_action_desc' => 'Select between "Purchase" to capture/invoice your order automatically or "Authorization" to capture/invoice manually.',
-    'wd_config_payolution_terms_url' => 'payolution URL',
+    'wd_config_payolution_terms_url' => 'Payolution URL',
```

Comment:
You updated all payolutions to capitalized, is this discussed with @hbrandl? As far as i know payolution is always written in lowercase.

Reply:
this was discussed in here: https://wd-cee.slack.com/archives/GFZKK7SCA/p1559573664024800?thread_ts=1559051638.019400&cid=GFZKK7SCA